champion.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, champion_name: str, item_names: Optional[List[str]] = None, l
         self.orig_base_stats = self.get_champion_base_stats(ALL_CHAMPION_BASE_STATS[champion_name])
         self.item_stats = self.inventory.get_items_total_stats(self.inventory.items)
         self.orig_bonus_stats = self.get_bonus_stats()
+        self.current_health = self.orig_base_stats["health"]
```

Comment:
as we anticipate the buff and debuff that a champion can encounter during a fight, we were thinking of having a base_stats and bonus_stats Stats object that are copies of their `orig` counterpart at the begining of the fight.
We would then alter the health, armor, ad etc inside those dynamic stats objects. 

Or do you think we should have the current stats directly as attribute of the champion

Reply:
Yeah I think we should just put stats in the champion directly